docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -94,37 +93,37 @@ In addition, you will need to know the following:
 - An <ProductConcept section="what-is-an-object" linkName="Object" />: represents an entity in the system. Users' relationships to it can be define through relationship tuples and the authorization model
-- A <ProductConcept section="what-is-a-relationship-tuple" linkName="Relationship Tuple" />: a grouping consisting of a user, a relation and an object stored in Auth <ProductName format={ProductNameFormat.ShortForm}/>
+- A <ProductConcept section="what-is-a-relationship-tuple" linkName="Relationship Tuple" />: a group consisting of a user, a relation, and an object stored in Auth <ProductName format={ProductNameFormat.ShortForm}/>
 - A <ProductConcept section="what-is-a-relationship" linkName="Relationship" />: <ProductName format={ProductNameFormat.ShortForm}/> will be called to check if there is a relationship between a user and an object, indicating that the access is allowed
 - [Direct Relationship Type Restrictions](../configuration-language.mdx#the-direct-relationship-type-restrictions): can be used to indicate direct relationships between users and objects
-- A <ProductConcept section="what-is-a-check-request" linkName="Check API Request" /> the Check API Request is used to check for relationships between users and objects
+- A <ProductConcept section="what-is-a-check-request" linkName="Check API Request" />: used to check for relationships between users and objects
 
 </details>
 
 <Playground />
 
 ## Step By Step
 
-To illustrate modeling Roles and Permissions in <ProductName format={ProductNameFormat.ShortForm}/>, we will use a trip booking system where you can have `owners` and/or `viewers` that can have more granular permissions such as adding bookings to a trip or viewing bookings on it.
+To illustrate modeling Roles and Permissions in <ProductName format={ProductNameFormat.ShortForm}/>, imagine a trip booking system that has `owners` and/or `viewers`. It can also have granular permissions like adding bookings to or viewing bookings on a trip.
 
-In order to represent this, we need to:
+To represent this, you need to:
 
-1. Understand how roles are related to direct relations for our trip booking system
-2. Adding implied relations to existing authorization model to define permissions for bookings
-3. <ProductConcept section="what-is-a-check-request" linkName="Checking" /> user roles and their permissions based on *relationship
-   tuples* for direct and implied relations
+1. Understand how roles are related to direct relations for the trip booking system
+2. Add implied relations to the existing authorization model to define permissions for bookings
+3. <ProductConcept section="what-is-a-check-request" linkName="Check" /> user roles and their permissions based on relationship
+   tuples for direct and implied relations
 
 ### 01. Understand How Roles Work Within Our Trip Booking System
 
```

Comment:
https://oktawiki.atlassian.net/wiki/spaces/DOCS/pages/2544472397/Document+Structure#Capitalization

Reply:
Per @rhamzeh, if we would like to reference a generalized "Best practices" ... I would refer here:

https://developers.google.com/style/headings?hl=en#heading-and-title-format

Essentially, we should be using sentence case for headers. Let me know if you have any questions. Thanks!